<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiClass</class>
 <widget class="QMainWindow" name="guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="m_wdgt_controls" native="true">
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Management controls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_cmbBx_devicesList"/>
       </item>
       <item>
        <widget class="QPushButton" name="m_btn_init">
         <property name="text">
          <string>init</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btn_start">
         <property name="text">
          <string>start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btn_stop">
         <property name="text">
          <string>stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>210</width>
            <height>515</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>1</x>
             <y>1</y>
             <width>191</width>
             <height>541</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="m_lbl_log">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="m_wdgt_video" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_btn_init</sender>
   <signal>clicked()</signal>
   <receiver>guiClass</receiver>
   <slot>f_v_handleButtonPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btn_start</sender>
   <signal>clicked()</signal>
   <receiver>guiClass</receiver>
   <slot>f_v_handleButtonPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btn_stop</sender>
   <signal>clicked()</signal>
   <receiver>guiClass</receiver>
   <slot>f_v_handleButtonPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cmbBx_devicesList</sender>
   <signal>textActivated(QString)</signal>
   <receiver>guiClass</receiver>
   <slot>f_v_handleButtonPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>f_v_handleButtonPress()</slot>
 </slots>
</ui>
